server/index.js

```diff
@@ -14,11 +14,24 @@ app.use(sessionMiddleware);
 app.use(express.json());
 
 app.get('/api/health-check', (req, res, next) => {
-  db.query(`select 'successfully connected' as "message"`)
+  db.query('select \'successfully connected\' as "message"')
     .then(result => res.json(result.rows[0]))
     .catch(err => next(err));
 });
 
+app.get('/api/enterRoom/:entryKey', (req, res, next) => {
```

Comment:
Change endpoint to '/api/rooms' since that's the name of the table we're accessing.

Reply:
got it! thanks!